realm/realm-library/src/main/cpp/CMakeLists.txt

```diff
@@ -91,7 +91,8 @@ use_realm_core(${build_SYNC} "${REALM_CORE_DIST_DIR}" "${CORE_SOURCE_PATH}")
 set(openssl_build_TYPE "release")
 # FIXME Read the openssl version from core when the core/sync release has that information.
 set(openssl_VERSION "1.0.2k")
-set(openssl_FILENAME "openssl-${openssl_build_TYPE}-${openssl_VERSION}-Android-${ANDROID_ABI}")
+set(openssl_BUILD_NUMBER "1")
```

Comment:
Perhaps add a comment that this number should match https://github.com/realm/openssl-android/pull/2/files#diff-5e3269174d70512b79d0b874838be93fR1 ?

Reply:
for the next core release, this will be removed since core will ship a openssl lib. no need to download anymore.